packages/app/src/components/inspector/FillRow.tsx

```diff
@@ -107,12 +124,24 @@ export default memo(function ColorFillRow({
     [onChangeGradientType],
   );
 
+  const handleSelectPatternSize = useCallback(
+    (value: PatternFillTypes) => {
+      if (onChangePatternFillType)
+        onChangePatternFillType(Sketch.PatternFillType[value]);
+    },
+    [onChangePatternFillType],
+  );
+
+  /**?
+   *
+   * Join all the onChange ?? :thinking:
```

Comment:
I think I prefer it like this, since otherwise we'll have to ignore certain types in the callbacks of border/shadow pickers